src/Defines.h

```diff
@@ -509,6 +512,317 @@ inline bool IsBlockFence(BLOCKTYPE a_BlockType)
 
 
 
+
+// net.minecraft.block.material.Material.WOOD
```

Comment:
Please remove all references to Minecraft code in all such comments to these functions.